public/src/main/java/com/revenuecat/purchases/models/PricingPhase.kt

```diff
@@ -28,4 +28,27 @@ data class PricingPhase(
      * [Price] of the [PricingPhase]
      */
     val price: Price
-) : Parcelable
+) : Parcelable {
+
+    /**
+     * Indicates how the pricing phase is charged for FINITE_RECURRING pricing phases
+     */
+    val paymentMode: PaymentMode?
+        get() {
+            // billingCycleCount is null for INFINITE_RECURRING or NON_RECURRING recurrence modes
```

Comment:
I'm wondering if for `INFINITE_RECURRING`, this should be `PAY_AS_YOU_GO` by default? I guess we should try to mimick iOS behavior for this.

Reply:
iOS only uses `PaymentMode` for `DiscountOffers` so I _think_ renaming this as `OfferPaymentMode` probably makes the most sense to align with iOS then 🤔